tests/unit/trace/test_tracer.py

```diff
@@ -34,7 +35,7 @@ def test_constructor_default(self):
         assert isinstance(tracer.sampler, AlwaysOnSampler)
         assert isinstance(tracer.exporter, print_exporter.PrintExporter)
         assert isinstance(tracer.propagator,
```

Comment:
newline!

Reply:
Sorry for these noisy format issues. Will try the vim automation as you suggested.